### PR TITLE
Fix nginx HTTP 413 error (request entity too large)

### PR DESCRIPTION
**Pull request by turicas**
nginx has the default `client_max_body_size` to `1MB`, but clients like `s3cmd` and `s4cmd` use a greater multipart size when uploading files to S3. This pull request uses Dokku's custom nginx config to set `client_max_body_size` according to the app environment variable `NGINX_MAX_REQUEST_BODY`.
This pull request fixes issue #7.

### CODE REVIEW

**Comment by mimischi** [comment]
Hi @turicas! Thanks for creating a pull request. I will try to review it over the weekend and get back to you!

File: README.md [hunk 1]
```diff
@@ -17,14 +17,16 @@ storage service. Read more at the [minio.io](https://www.minio.io/) website.
...
 
 ### Requirements
+
 * A working [Dokku host](http://dokku.viewdocs.io/dokku/getting-started/installation/)
 
 # Setup
 
-**Note:** We are going to use the domain `minio.example.com` for demonstration
-purposes. Make sure to replace it to your domain name.
+We are going to use the domain `minio.example.com` and dokku app `minio` for
```
Comment (mimischi) [inline README.md]:
```suggestion
We are going to use the domain `minio.example.com` and Dokku app `minio` for
```

File: README.md [hunk 2]
```diff
@@ -33,24 +35,40 @@ dokku apps:create minio
...
+> **Note:** if you do not set these keys, Minio will generate them during
+> startup and output them to the log (check if via `dokku logs minio`). You
+> will still need to set them manually.
+
+You'll also need to set other two environment variables:
+
+- `NGINX_MAX_REQUEST_BODY`: used in the custom `nginx.conf` for this Dokku app
+  to allow uploads up to 15MB to the HTTP server (if the file size is greater
+  than 15MB, `s3cmd` will split in 15MB parts).
+- `MINIO_DOMAIN`: used to tell minio the domain name being used by the server.
```
Comment (mimischi) [inline README.md]:
```suggestion
- `MINIO_DOMAIN`: used to tell Minio the domain name being used by the server.
```

**Review by mimischi — APPROVED** [review]
This looks good and I just confirmed that a) the deploy and b) upload of files >15MB via `s3cmd` both work.

Added two minor changes, but we do not have to do these before merging. Let me know if you want to accept these changes and then I'll merge!

Thanks again :tada:

**Comment by turicas** [comment]
Suggestions commited. :)

**Comment by mimischi** [comment]
Merged. Thanks a bunch! :)

**Pull request merged**